crates/modelardb_server/src/context.rs

```diff
@@ -222,13 +222,15 @@ impl Context {
     /// DataFusion. If the model tables could not be retrieved from the metadata Delta Lake or a
     /// model table could not be registered, return [`ModelarDbError`].
     pub async fn register_model_tables(&self) -> Result<(), ModelarDbError> {
-        let table_metadata_manager = &self.data_folders.local_data_folder.table_metadata_manager;
-
-        let model_table_metadata = table_metadata_manager
+        let model_table_metadata = self
```

Comment:
Maybe add a comment why `model_table_metadata` uses the `local_data_folder` while `table_metadata_manager` uses the `query_data_folder` as it is not clear.

Reply:
Added a comment that specifies that we use the local data folder since we don't want to register tables that the data sinks cannot write data to (since it uses the storage engine). The query data folder might contain more tables than the local data folder on start up but that should not be a problem since we retrieve and create the potentially missing tables from the manager right after registering tables and model tables locally.